src/bootstrap/src/core/build_steps/compile.rs

```diff
@@ -1965,6 +1965,16 @@ impl Step for Assemble {
             }
         }
 
+        {
+            // `llvm-strip` is used by rustc, which is actually just a symlink to `llvm-objcopy`,
+            // so copy and rename `llvm-objcopy` and create link for `rust-strip`
+            let src_exe = exe("llvm-objcopy", target_compiler.host);
+            let dst_exe = exe("rust-objcopy", target_compiler.host);
+            builder.copy_link(&libdir_bin.join(src_exe), &libdir_bin.join(dst_exe));
+            let dst_exe = exe("rust-strip", target_compiler.host);
```

Comment:
do you really need the symlink, since the strip command used in the compiler is `rust-objcopy`?

Reply:
Removed the symlink, this was a holdover from before I hit the "symlink can't be added to tarball" check.